query-creator/create-query/steps/aws.js

```diff
@@ -37,31 +36,34 @@ exports.ensureWorkgroup = async (user) => {
   return workgroup.WorkGroup;
 }
 
-exports.uploadDenormalizedResource = async (denormalizedResource, runnable, workgroup) => {
-  const key = `${runnable.type}-${runnable.id}`
+exports.uploadDenormalizedResource = async (queryId, denormalizedResource, workgroup) => {
   const s3 = new AWS.S3({apiVersion: '2006-03-01'});
   await s3.putObject({
     Bucket: process.env.OUTPUT_BUCKET,
     Body: JSON.stringify(denormalizedResource),
-    Key: `activity-structure/${key}/${key}-structure.json`
+    Key: `activity-structure/${queryId}/${queryId}-structure.json`
   }).promise()
+  // TODO: tie the uploaded file to the workgroup
 }
 
-exports.generateSQL = (runnable, resource, denormalizedResource) => {
+exports.generateSQL = (queryId, runnable, resource, denormalizedResource) => {
   const selectColumns = [];
   const whereClauses = [];
 
   Object.keys(denormalizedResource.questions).forEach(questionId => {
     const type = questionId.split(/_\d+/).shift();
     switch (type) {
       case "image_question":
-        selectColumns.push(`json_extract_scalar(answer, '$.image_url') AS ${questionId}`)
+        selectColumns.push(`json_extract_scalar(answer, '$.image_url') AS ${questionId}_image_url`)
+        selectColumns.push(`json_extract_scalar(answer, '$.text') AS ${questionId}_text`)
+        selectColumns.push("answer")
         break;
       case "open_response":
-        selectColumns.push(`kv1['${questionId}'] AS ${questionId}`)
+        selectColumns.push(`kv1['${questionId}'] AS ${questionId}_text`)
+        selectColumns.push(`submitted AS ${questionId}_submitted`)
         break;
       case "multiple_choice":
-        selectColumns.push(`activities.choices['${questionId}'][json_extract_scalar(kv1['${questionId}'], '$.choice_ids[0]')].content AS ${questionId}`)
+        selectColumns.push(`activities.choices['${questionId}'][json_extract_scalar(kv1['${questionId}'], '$.choice_ids[0]')].content AS ${questionId}_choice`)
```

Comment:
This looks like it won't handle a multiple select multiple_choice question correctly. It looks like it just takes the first first choice from choice_ids. 

I made separate stories for this, so getting it right doesn't hold up demoing this existing work:
https://www.pivotaltracker.com/story/show/173667973
https://www.pivotaltracker.com/story/show/173668043